### PR TITLE
Calculate and print job totals for scan and generate tasks

### PR DESCRIPTION
**Pull request by bnkai**
Calculates the number of new files we need to scan for the scan task and the total number of sprites,previews,transcodes,markers we need to generate.

### CODE REVIEW

File: pkg/manager/manager_tasks.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func (s *singleton) Scan(nameFromMetadata bool) {
 		}
 
 		total := len(results)
-		logger.Infof("Starting scan of %d files", total)
+		logger.Infof("Starting scan of %d files.%d New files found", total, s.neededScan(results))
```
Comment (WithoutPants) [inline pkg/manager/manager_tasks.go]: Add a space between `.` and `%d`.

File: pkg/manager/manager_tasks.go [hunk 2]
```diff
@@ -155,6 +155,8 @@ func (s *singleton) Generate(sprites bool, previews bool, markers bool, transcod
 			logger.Info("Stopping due to user request")
 			return
 		}
+		totalsNeeded := s.neededGenerate(scenes, sprites, previews, markers, transcodes)
+		logger.Infof("Generate Starting.Generating %d sprites %d previews %d markers %d transcodes", totalsNeeded.sprites, totalsNeeded.previews, totalsNeeded.markers, totalsNeeded.transcodes)
```
Comment (WithoutPants) [inline pkg/manager/manager_tasks.go]: Suggest improving levity and formatting: 
```
logger.Infof("Generating %d sprites, %d previews, %d markers, %d transcodes", totalsNeeded.sprites, totalsNeeded.previews, totalsNeeded.markers, totalsNeeded.transcodes)
```

**Comment by WithoutPants** [comment]
Seems to test ok. I'm wondering if this will add unnecessary overhead to scans/generates on slow links. It's been reported that scanning/generating is already slow on google drive for example, due to the amount of walking the directory that the scanner does to get the total workload. I'm concerned that this will slow it even further for such users.

**Comment by bnkai** [comment]
@WithoutPants the time for the exra directory scan is negligible for lans.It's almost the same time from pressing scan to seeing the message Starting scan from the "plain" scan version.It's only penalty is the doublestar glob time the rest is db and local fs. 
I don't have access to gdrive to test but i think the extra time needed won't be even close to the scan/generate of a single file from there and if you are using a slow link anyway it makes a lot more sense to know exactcly how many files you need to generate.
I'll ask around on discord to get an estimate on the that.

**Comment by bnkai** [comment]
Ok got feedback from Charlie on discord , even with a big collection on gdrive the needed time for doublestar is indeed negligible.

**Comment by WithoutPants** [comment]
I'm satisfied that this is good to merge. @Leopere

**Comment by Leopere** [comment]
Written by @bnkai and verified by @WithoutPants

**Pull request merged**